docs/versioned_docs/version-2.11.0/concepts/read-write-many.md

```diff
@@ -1,6 +1,11 @@
 ---
 id: rwm
 title: Provisioning Read-Write-Many (RWX) PVCs
+keywords: 
+  - Read-Write-Many
+  - RWX
+  - NFS server
```

Comment:
++
-OpenEBS
-OpenEBS community

Reply:
SEO keywords are the words and phrases that searchers enter into search engines and these help them land on the correct page. IMO, `OpenEBS community` will be the right keyword for only the community page.